src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,827 @@
++++
+title = "User Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 15
++++
+
+## Overview
+
+This guide is intended for users who have a Submariner environment set up and want to verify the installation and learn more about how to
+use Submariner and the main capabilities it provides. This guide assumes that there are two Kubernetes clusters,
+**cluster2** and **cluster3**, forming a clusterset, and that the Broker is deployed into a separate cluster **cluster1**.
+
+{{% notice tip %}}
+Make sure you have `subctl` [set up](../deployment/subctl/#installation). Regardless of how Submariner was deployed, `subctl` can be used
+for various verification and troubleshooting tasks, as shown in this guide.
+{{% /notice %}}
+
+{{% notice note %}}
+This guide focuses on a non-Globalnet Submariner deployment.
+{{% /notice %}}
+
+### 1. Validate the Installation
```

Comment:
May be all the validate installation stuff should be a different guide? troubleshooting?

It's weird if a usage guide starts by digging into the "low level" details :)

Reply:
I think that we need to separate between "verification" (i.e I just deployed Submariner and I want to know that it's up and running before I use it) and "troubleshooting" (i.e something went wrong). I agree that currently there is some overlap between the content I am proposing here and our "Troubleshooting Guide", but IMO we should fix/enhance the Troubleshooting Guide.

In any case, having accurate and up-to-date content is the most important thing; we can always move things around ;)